### PR TITLE
[CI:DOCS] Fix up documentation of the userns audit flag

### PR DESCRIPTION
**Pull request by rhatdan**
Add reference to the `containers` user in the /etc/subuid and
/etc/subgid files.

Fixes: https://github.com/containers/podman/issues/10906

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @giuseppe @mheon PTAL

File: docs/source/markdown/podman-create.1.md
Comment (mheon) [inline docs/source/markdown/podman-create.1.md:1091]: Should we mention root vs rootless usage here?

File: docs/source/markdown/podman-run.1.md
Comment (mheon) [inline docs/source/markdown/podman-run.1.md:1689]: Is this just in run, not create?

Comment (rhatdan) [reply docs/source/markdown/podman-run.1.md:1689]: I think it will work with create as well.  But I wanted to show examples and the userns generated, with the main goal showing they are unigue.

File: docs/source/markdown/podman-create.1.md
Comment (rhatdan) [reply docs/source/markdown/podman-create.1.md]: Added a comment about only being rootless.

File: docs/source/markdown/podman-run.1.md
Comment (rhatdan) [reply docs/source/markdown/podman-run.1.md]: Added a create example.

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
@mheon @TomSweeneyRedHat PTAL

**Review by vrothberg — APPROVED** [review]
LGTM

File: docs/source/markdown/podman-create.1.md
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-create.1.md:1089]:
```suggestion
**auto[:_OPTIONS,..._]**: automatically create a unique user namespace.
```

File: docs/source/markdown/podman-create.1.md
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-create.1.md:1091]:
```suggestion
The `--userns=auto` flag requires that the user name `containers` and a range of subordinate user ids that the Podman container is allowed to use be specified in the /etc/subuid and /etc/subgid files.  Example: `containers:2147483647:2147483648`. Podman allocates unique ranges of UIDs and GIDs from the `containers` group. The size of the ranges is based on the number of UIDs required in the image. The number of UIDs and GIDs can be overridden with the `size` option. The `auto` options currently do not work in rootless mode.
```

File: docs/source/markdown/podman-run.1.md
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-run.1.md:1167]: see comments from create

File: docs/source/markdown/podman-create.1.md
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-create.1.md:1095]: to fit the new manpage syntax, these should be only a single asterisk, except for the default value.

**Review by TomSweeneyRedHat — COMMENTED** [review]
Couple nits that need tending....

File: docs/source/markdown/podman-create.1.md
Comment (rhatdan) [reply docs/source/markdown/podman-create.1.md]: fixed

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
/lgtm
/hold
#until tests pass

**Pull request merged**
